dll/cpl/main/lang/ro-RO.rc

```diff
@@ -61,14 +65,14 @@ BEGIN
     CONTROL "", IDC_IMAGE_SWAP_MOUSE, "Static", SS_ICON | SS_CENTERIMAGE | WS_VISIBLE, 171, 14, 65, 57, WS_EX_STATICEDGE
     LTEXT "Bifați pentru interschimbarea funcției butoanelor principale ale șoricelului. Astfel butonul drept va fi cel folosit pentru selectare sau agățare și trasare.", -1, 10, 32, 156, 36
     GROUPBOX "Viteza pentru dublu-clic", -1, 5, 78, 236, 70
-    LTEXT "Executați dublu-clic pe dosarul mostră pentru a testa. Dacă dosarul nu reacționează, ajustați viteza pentru o execuție mai lentă.", -1, 10, 90, 156, 28
+    LTEXT "Executați dublu-clic pe dosarul mostră pentru a testa setările. Dacă dosarul nu se deschide sau nu se închide, ajustați viteza pentru o execuție mai lentă.", -1, 10, 90, 156, 28
```

Comment:
Please provide a screenshot of the romanian dialog IDD_PAGE_BUTTON. I want to see that this LTEXT control does not overflow the available area. It became longer, so I want to see the proof.

Reply:
![image](https://github.com/reactos/reactos/assets/113553683/82860c9e-008f-4f26-be8f-727b8e3b7be7)

Yes, you are right. I will add a new commit.